src/components/Alert/MuiAlert.tsx

```diff
@@ -0,0 +1,68 @@
+import { Palette } from '@mui/material';
+import Box, { BoxProps } from '@mui/material/Box';
+import React from 'react';
+import { tokens } from 'src/theme/tokens';
+import { CustomColors } from 'src/types';
+
+export interface IMuiAlertProps extends BoxProps {
+  icon?: React.ReactNode;
+  children: React.ReactNode;
+  variant?: Omit<CustomColors, 'accent' | 'secondary' | 'primary'>;
```

Comment:
do we need these here `'accent' | 'secondary' | 'primary'`?

Reply:
yes, there is no alert variants for those
It's the same as me creating a new color variants for the alert alone